content/en/blog/2025/slack-workspace-changes.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Action Required - Slack Workspace Changes are Coming Friday, June 20
+linkTitle: Slack Workspace Changes
+date: 2025-06-17
+author: >-
+  OpenTelemetry Authors
```

Comment:
I think that it is ok to put your name, but if you want something more team-oriented, then how about (regardless you can drop the '>-\n' :):

```suggestion
author: OpenTelemetry maintainers
```